.eslintrc.json

```diff
@@ -9,6 +9,7 @@
     "graphql/named-operations": ["error"],
     "graphql/capitalized-type-name": ["error"],
     "graphql/no-deprecated-fields": ["warn"],
+    "node/no-unpublished-import": ["warn"],
```

Comment:
Would be great to not have the `warning  "styled-icons" is not published  node/no-unpublished-import`, but also we should not silent this rule. How can we achieve that?

Reply:
Uhmm, I think https://github.com/opencollective/opencollective/issues/1832#issuecomment-474639542 is a good idea. We have too many errors to control individually.

Reply:
tl;dr I have no idea